spec/grape/integration/rack_spec.rb

```diff
@@ -31,4 +31,26 @@
       input.unlink
     end
   end
+
+  context 'when the app is mounted' do
+    def app
```

Comment:
This pollutess the global namespace, put it into a namespace itself?

Reply:
In what way @dblock ? I am under the impression that methods defined in a context are contained to that context?

Reply:
Okay, this is news to me. Since when does it work as one would expect?!

```ruby
require 'rspec'

describe 'Context' do
  def m1
    'm1'
  end

  it 'is m1' do
    expect(m1).to eq 'm1'
  end

  context 'c1' do
  	def m2
	  'c1:m2'
  	end

    it 'is m1' do
      expect(m1).to eq 'm1'
    end

    it 'is m2' do
      expect(m2).to eq 'c1:m2'
    end
  end

  context 'c2' do
	it 'is m2' do
	  expect(m2).to eq 'c1:m2'
    end
  end
end
```

```
  1) Context c2 is m2
     Failure/Error: expect(m2).to eq 'c1:m2'
     
     NameError:
       undefined local variable or method `m2' for #<RSpec::ExampleGroups::Context::C2 "is m2" (./spec.rb:27)>
     # ./spec.rb:28:in `block (3 levels) in <top (required)>'
```

My bad. Ignore me.